integration_tests/src/main/python/array_test.py

```diff
@@ -34,6 +34,40 @@
 array_item_test_gens = array_gens_sample + [array_all_null_gen,
     ArrayGen(MapGen(StringGen(pattern='key_[0-9]', nullable=False), StringGen(), max_length=10), max_length=10)]
 
+
+_non_neg_zero_float_special_cases = [
```

Comment:
Is there an issue filed about -0.0? Can we reference it here so there is some kind of tracking?

Reply:
It's this one: https://issues.apache.org/jira/browse/SPARK-39845. I will also add a reference in the code.